### ISSUE TITLE
Courseplay is not detected on FS 15 (v1.1)

### ISSUE THREAD

**Issue by imicomputer** [Courseplay/courseplay#1161]
I've tried to install this on Win7, following the installation guide to put the ZZZ_courseplay.zip file In the FS15's mods folder along with few others mods which are working. And even tried to download the dev version, unzipped in, re-zipped what's inside the course_master folder. But nothing's seams to work. No Right-mouse button shortcut showed, no nothing from the courseplay.
Even tried to remove all the other mods I've.

What have I done wrong?

**Comment by gui7545** [comment]
Can you put your log here : https://gist.github.com/ ? and send the link ?

**Comment by imicomputer** [comment]
I've found the issue by looking the log file as you said @gui7545.

Found out this line that saying the Mod's Description Version is not supported, and tried to look it in the modDesc.xml file. The problematic tag was <modDesc descVersion="22"> and compare it with a couple other working mods, it showed "21" instead of "22". After replacing the number the mod showed up in the Game's mod list.

I'm currently using FS 15 v1.1. (fyi, this seams no problem with v1.2 Russian version, need confirmation on that)
